firefox-ios/Client/Coordinators/Browser/BrowserDelegate.swift

```diff
@@ -31,6 +31,12 @@ protocol BrowserDelegate: AnyObject {
         toastContainer: UIView
     )
 
+    /// Returns a tool which can be used to get a snapshot of the homepage
+    func homepageScreenshotTool() -> Screenshotable?
+
+    /// Hides or show the homepage.
+    func setHomepageVisibility(isVisible: Bool)
```

Comment:
This is only used for taking screenshots, correct? Maybe we should add the reasoning behind this method to the documentation too?